### PR TITLE
[`#36`] Add README badge statuses

### LINKED ISSUE #36: Add status CI badges to the `README`
Add status badges to the README:
https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

### PR DESCRIPTION
**Pull request by 0101coding**
Add README status badges.

Resolves:
* https://github.com/John15321/rust-pip/issues/36

### CODE REVIEW

**Comment by John15321** [comment]
Tell me explicitly if i should review it

**Comment by 0101coding** [comment]
> Tell me explicitly if i should review it

You can check it on my repo. That should suffice.

**Comment by 0101coding** [comment]
YES

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -9,6 +9,12 @@ env:
     CARGO_TERM_COLOR: always
 
 jobs:
+    lint-markdowns:
+        name: Lint markdown files
+        runs-on: ubuntu-latest
+        steps:
+            - name: Lint markdown files 
+              uses: actionshub/markdownlint@main
```
Comment (John15321) [inline .github/workflows/build.yml:17]: Add a warning to ignore. We know that in the Changelog there will be multiple same headers which will cause it to throw an error
https://github.com/markdownlint/markdownlint/issues/175

**Comment by John15321** [comment]
Now I can see that there are two PRs with the same changes:
* https://github.com/John15321/rust-pip/pull/46

Delete creating the CI job in this PR

**Pull request closed without merging**